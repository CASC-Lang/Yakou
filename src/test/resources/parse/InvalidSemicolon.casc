class InvalidSemicolon

impl InvalidSemicolon {
    comp {
        a := 1;
    }
}