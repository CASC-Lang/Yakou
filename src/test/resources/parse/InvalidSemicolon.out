error: Redundant semicolon after statement
--> ./InvalidSemicolon.casc:5:14
4 |     comp {
5 |         a := 1;
                  ^= hint: Remove this semicolon
6 |     }